### PR TITLE
Ingest FasterXML repos using 2.17 branch

### PR DESCRIPTION
**Pull request by knutwannheden**

### CODE REVIEW

**Review by timtebeek — APPROVED** [review]

File: repos.csv [hunk 1]
```diff
@@ -2994,43 +2994,42 @@ scmHost,repoName,repoBranch,mavenTool,gradleTool,jdkTool,repoStyle,repoBuildActi
 ,FasterXML/StaxMate,master,maven,,,,,,
 ,FasterXML/TransiStore,master,maven,,,,,,
 ,FasterXML/aalto-xml,master,maven,,,,,,
-,FasterXML/jackson-annotations,2.14,,,,,,,
-,FasterXML/jackson-benchmarks,2.13,,,,,,,
-,FasterXML/jackson-core,2.14,maven,,,,,,
-,FasterXML/jackson-databind,2.14,,,,,,,
+,FasterXML/jackson-annotations,2.17,,,,,,,
+,FasterXML/jackson-benchmarks,2.15,,,,,,,
```
Comment (knutwannheden) [inline repos.csv:2998]: Here 2.15 is indeed the latest release branch.

File: repos.csv [hunk 2]
```diff
@@ -2994,43 +2994,42 @@ scmHost,repoName,repoBranch,mavenTool,gradleTool,jdkTool,repoStyle,repoBuildActi
...
+,FasterXML/jackson-jaxrs-providers,2.17,maven,,,,,,
 ,FasterXML/jackson-jdk11-compat-test,master,maven,,,,,,
 ,FasterXML/jackson-jdk17-compat-test,main,maven,,,,,,
 ,FasterXML/jackson-jdk6-compat-test,master,maven,,,,,,
-,FasterXML/jackson-jr,2.14,maven,,,,,,
+,FasterXML/jackson-jr,2.17,maven,,,,,,
 ,FasterXML/jackson-module-afterburner,master,maven,,,,,,
 ,FasterXML/jackson-module-jaxb-annotations,2.8,maven,,,,,,
 ,FasterXML/jackson-module-jsonSchema,master,maven,,,,,,
-,FasterXML/jackson-module-kotlin,2.14,maven,,,,,,
```
Comment (knutwannheden) [inline repos.csv:3029]: We probably don't need both.

File: repos.csv [hunk 3]
```diff
@@ -2994,43 +2994,42 @@ scmHost,repoName,repoBranch,mavenTool,gradleTool,jdkTool,repoStyle,repoBuildActi
...
 ,FasterXML/TransiStore,master,maven,,,,,,
 ,FasterXML/aalto-xml,master,maven,,,,,,
-,FasterXML/jackson-annotations,2.14,,,,,,,
-,FasterXML/jackson-benchmarks,2.13,,,,,,,
-,FasterXML/jackson-core,2.14,maven,,,,,,
-,FasterXML/jackson-databind,2.14,,,,,,,
+,FasterXML/jackson-annotations,2.17,,,,,,,
+,FasterXML/jackson-benchmarks,2.15,,,,,,,
+,FasterXML/jackson-core,2.17,maven,,,,,,
+,FasterXML/jackson-databind,2.17,,,,,,,
```
Comment (timtebeek) [inline repos.csv:3000]: Did notice that there's also for instance a https://github.com/FasterXML/jackson-databind/tree/master, which has the same commit as the 2.17 main branch. Not sure what their flow is specifically, but if they consistently also maintain `master` or `main` we might want to switch to that.

Comment (knutwannheden) [reply repos.csv:3000]: I am also unsure. The 2.17 branch is their default branch in GitHub. But the next release is indeed being developed on master. I am unsure if the next release will be 3.0 or 2.18.

**Pull request merged**